modules/cnf-topology-aware-lifecycle-operator-autocreate-cgu-cr-ztp.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="talo-precache-autocreated-cgu-for-ztp_{context}"]
+= About the auto-created ClusterGroupUpgrade CR for ZTP
+
+The TALO has a controller called `ManagedClusterForCGU` that monitors the `Ready` state of the `ManagedCluster` CRs on the hub cluster and creates the `ClusterGroupUpgrade` CRs for ZTP (zero touch provisioning).
+
+For any managed cluster in the `Ready` state without a "ztp-done" label applied, the `ManagedClusterForCGU` controller automatically creates a `ClusterGroupCluster` CR in the `ztp-install` namespace with its associated {rh-rhacm} policies that are created during the ZTP process. The TALO then remediates the set of configuration policies that are listed in the auto-created `ClusterGroupUpgrade` CR to push the configuration CRs to the managed cluster.
```

Comment:
Are we sure `ClusterGroupCluster` is correct?

Reply:
@sudomakeinstall2 Good catch, that is not correct. Fixed